packages/esm-patient-registration-app/src/patient-registration/field/field.scss

```diff
@@ -55,6 +55,17 @@
   justify-content: flex-start;
 }
 
+.dobField {
+  :global(.cds--date-picker-input__wrapper) {
+    max-width: 8.125rem;
+
+    :global(.cds--date-picker__icon) {
+      right: unset;
+      left: 10.5rem;
+    }
+  }
+}
```

Comment:
overriding styles for the date input

Reply:
Why are we having the overrides?

Reply:
I wanted to make the date input quite small as the max-characters it can support.  Also to align it with the small size of the inputs. If you compare with the identifier inputs below
![image](https://github.com/openmrs/openmrs-esm-patient-management/assets/104269786/2f04d7d2-c8eb-47d4-ae0c-068ebb7a142d)